src/frequenz/sdk/timeseries/battery_pool/battery_pool.py

```diff
@@ -353,10 +353,11 @@ def soc(self) -> MetricAggregator[Sample[Percentage]]:
         ```
         working_batteries: Set[BatteryData] # working batteries from the battery pool
 
-        soc_scaled = max(
+        # TODO what bound is used?
```

Comment:
do you want to keep this TODO message? just checking...

Reply:
no, classic oversight from my side :).